charts/openstack-cluster/values.yaml

```diff
@@ -97,6 +98,10 @@ clusterNetworking:
     # The CIDR to use if creating a cluster network
     # This is only used if neither of networkFilter and subnetFilter are given
     nodeCidr: 192.168.3.0/24
+    # List of nameserver IPs to use when creating a cluster network.
+    # This is only used if neither of networkFilter and subnetFilter are given
+    dnsNameservers: []
```

Comment:
Can we default this to `null` as before, instead of `[]`?

Reply:
Annoying style question:

Functionally this is no different (`default` deals with empty lists as if they are `null`) but I felt it provided a hint to the user of the chart as to the type of field this is expected to be.

As before, happy to make the change just interested in why :) It would match the rest of the file better, so that's reason enough perhaps.

Reply:
I was just going for consistency with the previous default, TBH. Not specifying it means "use the network default DNS servers", so I thought null was a better default than `[]` at the time. I'm not that bothered though.